packages/design-system-react/src/components/index.ts

```diff
@@ -11,6 +11,13 @@ export type { AvatarBaseProps } from './AvatarBase';
 export { AvatarFavicon, AvatarFaviconSize } from './AvatarFavicon';
 export type { AvatarFaviconProps } from './AvatarFavicon';
 
+export {
+  AvatarGroup,
+  AvatarGroupSize,
+  AvatarGroupVariant,
+} from './AvatarGroup';
+export type { AvatarGroupProps } from './AvatarGroup';
```

Comment:
Should we be exporting these from the root `index.ts` as well?

Reply:
![Screenshot 2025-04-17 at 10 27 04 AM](https://github.com/user-attachments/assets/706bde96-bca3-44f5-8639-abe74fb88942)